modules/ROOT/pages/guided-setup.adoc

```diff
@@ -25,12 +25,54 @@ This may be the US control plane, the EU control plane, the MuleSoft Government
 +
 If your Anypoint account has the organization administrator role, the guided setup tool can create the connected app for you.
 +
-Otherwise, create the connected app or ask an organization administrator to create it, then provide the app details so the guided setup can perform the authentication and authorization.
+Otherwise, create the connected app or ask an organization administrator to create it with these steps:
++
+.. Create a key pair in Salesforce.
++
+This key pair is created in Salesforce and used for communication between Salesforce and Anypoint Platform.
++
+... Click *Setup* > *Security* > *Certificate and Key Management* > *Create Self-Signed Certificate*.
+... Set *Label/Name* to `Anypoint`.
+... Disable *Exportable Private Key*.
+... Set *Key Size* to 2048 or larger.
+... *Save*.
+... Click *Download Certificate*.
++ 
+The public key is saved on your local machine and is required in the next step to create the connected app in Anypoint Platform.
++
+.. Create a connected app in Anypoint Platform.
+... Click *Access Management* > *Connected Apps* > *Create App*.
+... Choose a name.
```

Comment:
Is the user typing in a name or selecting a name from a list?

Reply:
typing- rewrote to clarify